dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -287,8 +305,11 @@ private void validateDbColumn(Field field) throws DotDataException {
     if(field.dbColumn()==null){
       throw new DotDataValidationException("Unable to save field with a null dbColumn field.field_contentlet:" + field, "message.field.dbcolumn.incorrect");
     }
-    
-    if( !field.dbColumn().matches("(system_field|(text|float|bool|date|text_area|integer)[0-9]+)")){
+	String dbColumnNameRegex = "(system_field|(text|float|bool|date|text_area|integer)[0-9]+)";
```

Comment:
You might want to just handle this upstream where this exception is caught.  We already have all the logic about isNew and all that there.  It might be cleaner just to make this log a debug.

https://github.com/dotCMS/core/blob/a0ad23e11f0bbb98d3a4340d1069086c72224637/dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java#L171